vms/event/views.py

```diff
@@ -115,6 +116,7 @@ def list_sign_up(request, volunteer_id):
                 )
     else:
         event_list = get_events_ordered_by_name()
+        event_list = remove_empty_events(event_list, volunteer_id)
         return render(
             request,
             'event/list_sign_up.html',
```

Comment:
Changes in /event/services.py, /event/views.py should not come in this PR. This work is to fix #175. So should be present only in PR #176.